packages/client/utils/draftjs/completeEntity.tsx

```diff
@@ -80,29 +94,60 @@ const completeEntity = (
   const {keepSelection} = options
```

Comment:
-1 I think since this function is used also by other functionality (like adding emojis) and the omitting space logic is limited to link handling, I think we should move the logic to https://github.com/ParabolInc/parabol/blob/7a409665830e2b00166f3306e611ad31759f0b32/packages/client/components/EditorLinkChanger/EditorLinkChangerDraftjs.tsx#L39
What do you think?

Then we might be able to basically do:
* store current selection
* shrink selection to text only part (might collapse into just the input cursor)
* replace with link
* expand selection to include previous whitespace

What do you think?

Reply:
Ok, thanks!

Reply:
Done